Builds/containers/gitlab-ci/pkgbuild.yml

```diff
@@ -131,6 +132,73 @@ dpkg_build:
   script:
     - . ./Builds/containers/gitlab-ci/build_package.sh dpkg
 
+#########################################################################
+##                                                                     ##
+##  stage: sign_packages                                               ##
+##                                                                     ##
+##  build packages using containers from previous stage.               ##
+##                                                                     ##
+#########################################################################
+
+rpm_sign:
+  stage: sign_packages
+  dependencies:
+    - rpm_build
+  image:
+    name: centos:7
+    # <<: *dind_param
+  before_script:
+  - |
+    # Make sure GnuPG is installed
+    yum -y install gnupg rpm-sign
+    # checking GPG signing support
+    if [ -n "$GPG_KEY_B64" ]; then
+      echo "$GPG_KEY_B64"| base64 -d | gpg --batch --no-tty --allow-secret-key-import --import -
+      unset GPG_KEY_B64
+      export GPG_PASSPHRASE=$(echo $GPG_KEY_PASS_B64 | base64 -di)
+      unset GPG_KEY_PASS_B64
+      export GPG_KEYID=$(gpg --with-colon --list-secret-keys | head -n1 | cut -d : -f 5)
+    else
+      echo -e "\033[0;31m****** GPG signing disabled ******\033[0m"
+      exit 1
+    fi
+  artifacts:
+    paths:
+      - build/rpm/packages/
+  script:
+    - ls -alh build/rpm/packages
+    - . ./Builds/containers/gitlab-ci/sign_package.sh rpm
+
+dpkg_sign:
+  stage: sign_packages
+  dependencies:
+    - dpkg_build
+  image:
+    name: ubuntu:19.04
```

Comment:
You really want/need to build with a version of Ubuntu that's going to be EOL in ~2 months (https://wiki.ubuntu.com/Releases)? Did you maybe mean 18.04 instead? Or 19.10?

Reply:
I agree with @MarkusTeufelberger I'd rather use 18.04 here.

Reply:
we're using the container to run one command in this case (`dpkg-sig`) so I think we can make this a TODO item to switch to one of the LTS releases. In the meantime, I'm not losing any sleep over this.

Reply:
I'll cut a new PR to move it to the LTS release, but as @mellery451 said I'm just using it to get _one_ package that we're running, not for actual ubuntu platform testing.

Reply:
resolved via https://github.com/ripple/rippled/pull/3147